src/main/scala/nl.knaw.dans.easy.bagindex/components/IndexBagStoreComponent.scala

```diff
@@ -44,13 +44,16 @@ trait IndexBagStoreComponent extends DebugEnhancedLogging {
      * @return `Success` if the indexing was successful; `Failure` otherwise
      */
     def indexBagStore()(implicit connection: Connection): Try[Unit] = {
-      trace(())
+      logger.info(s"indexing bag-stores ${ bagStore.baseDirs.mkString("{", ", ", "}") }")
+      logger.info("clearing index")
```

Comment:
"clearing index" and further seems to me more like a debug statement.

Reply:
👍 I'll fix that